### ISSUE TITLE
Some studies show up twice in the search results list

### ISSUE THREAD

**Issue by Jeremy37** [opentargets/genetics#502]
E.g. search for either of these:
Schwartzentruber
Marioni
You will see that the single study present for these authors shows up twice in the search results droplist.
This occurs whether you search on the author name or on the study ID (e.g. GCST90012877).
I notice that these studies both have two EFO IDs, but then there are other studies with 2 EFO IDs which don’t show up as duplicates.
These didn’t show as duplicates in genetics portal R5.
![image](https://user-images.githubusercontent.com/11073896/141766586-8a5fda15-21d2-4331-b401-46d3319e6c05.png)

**Comment by mkarmona** [comment]
@ireneisdoomed this might be related to the fact that the last file contains EFO codes exploded
```
@ val efos = spark.read.parquet("trait_efo-2021-11-09.parquet") 
efos: DataFrame = [study_id: string, trait_reported: string ... 2 more fields]

@ efos.filter($"study_id" === "GCST90012877").show(false) 
+------------+------------------------------------------------------------+----------------------+-------------+
|study_id    |trait_reported                                              |trait_category        |trait_efos   |
+------------+------------------------------------------------------------+----------------------+-------------+
|GCST90012877|Alzheimer's disease or family history of Alzheimer's disease|measurement           |[EFO_0009268]|
|GCST90012877|Alzheimer's disease or family history of Alzheimer's disease|nervous system disease|[EFO_0000249]|
+------------+------------------------------------------------------------+----------------------+-------------+
```

**Comment by ireneisdoomed** [comment]
The extraction of the most relevant therapeutic area was not taken into account that a study/trait could be mapped to multiple EFOs, and therefore present multiple relevant TAs.

The error has been fixed on [a52d633](https://github.com/opentargets/genetics-v2d-data/commit/a52d6333da98f9b50c0efc5817717c97648f65fc). The grouping is now done at an earlier stage, before assigning the TA.

**The fixed LUT is here:** `gs://genetics-portal-dev-staging/v2d/210922/trait_efo-2021-11-16.parquet`

Once the study is updated, the error should be fixed.
